src/charts/UsageBySession.js

```diff
@@ -102,6 +125,24 @@ class UsageBySession extends Component {
               </TableRow>
             ))}
           </TableBody>
+          <TableFooter style={{ padding: "50px" }}>
+            <TableRow>
+              <TableCell>
+                Showing {this.state.sessionsInfo.length} of {this.state.totalCount} results.
```

Comment:
Minor suggestion on the code style: You already have an instance `sessionsInfo` that you can use from the line 99. Actually you can create instances also for loadingMore, endCursor and totalCount in order to avoid writing "this.state".